### PR TITLE
eslint and prettier configured and mongodb is set

### PR DESCRIPTION
**Pull request by khurmatillo23**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -10,9 +10,11 @@
       "license": "ISC",
       "dependencies": {
         "body-parser": "^1.19.2",
+        "cors": "^2.8.5",
```
Comment (Morphine-medicine) [inline package-lock.json:13]: Package-lock.json should be removed from repository and added to gitignore

**Review by Morphine-medicine — CHANGES_REQUESTED** [review]
Remove package-lock json

**Comment by khurmatillo23** [comment]
I added package-lock.json to gitignore file but why should I do like this?
And should I delete package-lock.json in git repository?

File: package-lock.json (hunk 1 above)
Comment (khurmatillo23) [reply package-lock.json:13]: I added package-lock.json to gitignore

**Comment by Morphine-medicine** [comment]
> I added package-lock.json to gitignore file but why should I do like this? And should I delete package-lock.json in git repository?

Yep, you should delete it. You need it because pakcage-lock.json is generated on npm i, so newest version of packages will be used, so it can be optimized

**Review by Morphine-medicine — APPROVED** [review]

**Pull request merged**